master/include/embedded/digitalReceiver.hpp

```diff
@@ -110,12 +107,9 @@ inline void DigitalReceiver::readAsmsSwitch() {
         digitalData->asms_on = false;
 }
 
-inline void DigitalReceiver::readAatsSwitch() {
-    aats_switch.update();
-    if (aats_switch.pressed())
-        digitalData->aats_on = true;
-    else
-        digitalData->aats_on = false;
+inline void DigitalReceiver::readTSState() {
+    // TS is on if SDC is closed (SDC STATE PIN AS HIGH)
```

Comment:
Comment may need a fix, TS being on is not the same as AATS being pushed in

Reply:
rename this to sdc_open (variable already in system data as SDC_State_open or something. this is to make following state logic and checks easier (initial checkup sequence, state machine diagram etc)